vlib/v/checker/tests/unwrapped_result_infix_err.vv

```diff
@@ -0,0 +1,17 @@
+fn f(k string) !bool {
+	return true
+}
+
+fn g() ! {
+	m := {
+		'1': true
+	}
```

Comment:
the map here can be removed too, only the key '1' from it is needed below

Reply:
Noted

Reply:
Done